esbuild.config.prod.mjs

```diff
@@ -0,0 +1,92 @@
+import * as esbuild from 'esbuild'
+import cssModulesPlugin from 'esbuild-css-modules-plugin'
+import fs from 'fs'
+
+const buildsConfig = [
+  {
+    format: 'esm',
+    outfile: 'dist/react-tooltip.esm.js',
+    minify: false,
+  },
+  {
+    format: 'cjs',
+    outfile: 'dist/react-tooltip.cjs.js',
+    minify: false,
+  },
+  {
+    format: 'iife',
+    outfile: 'dist/react-tooltip.iife.js',
+    minify: false,
+  },
+  {
+    format: 'esm',
+    outfile: 'dist/react-tooltip.js', // for styles be exported as `react-tooltip.css`
+    minify: false,
+  },
+  {
+    format: 'esm',
+    outfile: 'dist/react-tooltip.esm.min.js',
+    minify: true,
+  },
+  {
+    format: 'cjs',
+    outfile: 'dist/react-tooltip.cjs.min.js',
+    minify: true,
+  },
+  {
+    format: 'iife',
+    outfile: 'dist/react-tooltip.iife.min.js',
+    minify: true,
+  },
+  {
+    format: 'esm',
+    outfile: 'dist/react-tooltip.min.js',
+    minify: true,
+  },
+]
+
+const builds = []
+
+buildsConfig.forEach(({ format, outfile, minify }) => {
+  builds.push(
+    esbuild.build({
+      entryPoints: ['./src/index.tsx'],
+      bundle: true,
+      outfile,
+      format,
+      treeShaking: true,
+      minify,
+      sourcemap: true,
+      external: ['react', 'react-dom', 'prop-types'],
+      plugins: [
+        cssModulesPlugin({
+          // inject: true,
+          v2: true,
+          v2CssModulesOption: {
+            pattern: `react-tooltip__[local]_[hash]`,
+          },
+        }),
+      ],
+    }),
+  )
+})
+
+await Promise.all(builds)
```

Comment:
`Promise.allSettled()` would fail silently if any errors ocurred.

Reply:
That's true!

Thanks for this fix!